Float.Core/Compatibility/DeviceProxy.cs

```diff
@@ -32,7 +32,7 @@ internal static async Task InvokeOnMainThreadAsync(Action action)
         {
             try
             {
-                var method = typeof(Device).GetMethod("InvokeOnMainThreadAsync", BindingFlags.Static | BindingFlags.Public);
+                var method = typeof(Device).GetMethod("InvokeOnMainThreadAsync", new[] { typeof(Action) });
```

Comment:
@EBusch I think this avoids the conflict because it _might_ avoid searching for the [generic](https://docs.microsoft.com/en-us/dotnet/api/xamarin.forms.device.invokeonmainthreadasync?view=xamarin-forms#xamarin-forms-device-invokeonmainthreadasync-1(system-func((-0)))) variety of this method.  I'm a bit surprised this compiled before :shrug: